src/samarium/core.py

```diff
@@ -127,7 +127,7 @@ def run(
     Runtime.quit_on_error = quit_on_error
     code = Transpiler(tokenize(code), reg).transpile().output
     if load_template:
-        code = readfile(f"{os.path.dirname(__file__)}/template.py").replace(
+        code = readfile(Path(__file__).resolve().parent / 'template.py').replace(
```

Comment:
```suggestion
        code = readfile(Path(__file__).resolve().parent / "template.py").replace(
```

Also, now the typehint for `readfile` (`utils.py`) is incorrect. Btw I'm not familiar with `pathlib`, but I believe there is a way to read files without using a context manager—so maybe the entire function could be replaced with some `pathlib` operation?

Reply:
Great point, and yes: [`Path.open`](https://docs.python.org/3/library/pathlib.html#pathlib.Path.open).